scripts/compile.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# check versions
```

Comment:
Drive-by: constrain these versions (to match the ones in `.github/workflows/lint.yaml`) to produce consistent outputs